src/main/java/com/intuit/graphql/orchestrator/datafetcher/ServiceDataFetcher.java

```diff
@@ -2,12 +2,11 @@
 
 import com.intuit.graphql.orchestrator.schema.ServiceMetadata;
 import graphql.GraphQLContext;
-import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 import lombok.Getter;
 
 @Getter
-public class ServiceDataFetcher implements DataFetcher {
+public class ServiceDataFetcher implements ServiceAwareDataFetcher {
```

Comment:
same... if possible, would be good to have `ServiceAwareDataFetcher<CompletableFuture<DataFetcherResult>>`